packages/cozy-konnector-libs/src/libs/categorization/index.js

```diff
@@ -11,6 +11,12 @@ const logger = require('cozy-logger')
 
 const log = logger.namespace('categorization')
 
+/**
+ * @typedef CreateCategorizerOptions
+ * @property {boolean} useGlobalModel Whether to use the globally trained model
+ * @property {Function} fetchTransactions A custom training transaction fetcher
```

Comment:
nit: `fetchTransactions` sounds more boolean than func. Maybe a name `customTransactionFetcher` would make what it does more obvious: it's a fetcher function, and custom (implying it's optional).